packages/artifactor/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "sourceMap": true,
     "outDir": "dist",
     "baseUrl": ".",
-    "lib": ["es2017"],
+    "lib": ["es2017","dom"],
```

Comment:
I don't understand the point of this, since obviously there's no actual DOM here... you say it's about getting rid of some warning?  What warning is that, exactly?

Reply:
Oh, I see -- when using `console`.  I failed to parse that correctly at first.  Is there some way to tell it about Node instead of telling it about the DOM?

Reply:
I have removed that change, as "dom" is for browser apps mainly.

When I was working with it, a warning was showing, but it shouldn't have since @types/node package was already there. I have removed it... and the warning is gone anyway.

Reply:
I think I was getting the issue before bootstrapping the project and @types/node not being installed yet. I blindly followed vscode suggestion, but the issue is not there with a bootstrapped project.